userCode/groundwater/wells.py

```diff
@@ -252,7 +253,10 @@ def fetch_wells():
         }
         encoded_params = urlencode(params)
         logging.debug(f"Fetching {base_url}&{encoded_params}")
-        cache = ShelveCache()
+        # We want to cache the results of this API call since it is expensive
+        isSunday = datetime.datetime.now().weekday() == 6
+        # skip caching and always fetch once a week, otherwise cache it so it doesn't take forever
+        cache = ShelveCache(skip_caching_in_prod=True if isSunday else False)
         response, status = cache.get_or_fetch(
             f"{base_url}&{encoded_params}", force_fetch=False
```

Comment:
Why are we creating a new variable for this?
```suggestion
        cache = ShelveCache()
        # We want to cache the results of this API call since it is expensive
        isSunday = datetime.datetime.now().weekday() == 6
        response, status = cache.get_or_fetch(
            f"{base_url}&{encoded_params}", force_fetch=isSunday
```